app/src/main/java/com/android/brewr/ui/overview/EditJourney.kt

```diff
@@ -172,6 +168,7 @@ fun EditJourneyScreen(
                             coffeeRate = coffeeRate,
                             date = selectedDate)
                     listJourneysViewModel.updateJourney(updatedJourney)
```

Comment:
Very nice that you have fixed the part when we edit the journey, it updates it immediately.